packages/esify/lib/eslint.js

```diff
@@ -0,0 +1,16 @@
+var exec = require('child_process').exec;
+var path = require('path');
+var fs = require('fs');
+
+module.exports = function runESLint(details) {
+  return new Promise(function(resolve) {
```

Comment:
It was actually a linting environment thing that needed to be fixed. Thanks for catching that.